cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/DraftHandler.java

```diff
@@ -90,8 +90,7 @@ public void listDrafts(HttpServiceRequest request, HttpServiceResponder responde
       }
 
       SortRequest sortRequest = new SortRequest(sortBy, sortOrder);
-      String userId = request.getUserId();
-      userId = userId == null ? "" : userId;
+      String userId = "";
```

Comment:
Add comment explaining why done this way? e.g. to allow draft visible for all users within the same namespace.